src/Sylius/Bundle/CoreBundle/Form/Type/Product/ProductVariantType.php

```diff
@@ -88,39 +77,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'empty_value' => '---',
                 'label' => 'sylius.form.product_variant.tax_category',
             ])
-            ->add('pricingCalculator', 'sylius_price_calculator_choice', [
-                'label' => 'sylius.form.shipping_method.calculator',
+            ->add('channelPricings', CollectionType::class, [
+                'entry_type' => ChannelPricingType::class,
```

Comment:
I've done it this way due to upcoming upgrade to Symfony3 - or can it be done differently?